lib/src/view.rs

```diff
@@ -309,6 +309,17 @@ impl View {
         self.data.tags.get(name).flatten()
     }
 
+    /// Iterates local tag `(name, target)`s matching the given pattern. Entries
+    /// are sorted by `name`.
+    pub fn local_tags_matching<'a: 'b, 'b>(
```

Comment:
nit: `tags_matching()` (without `local_`)? We don't have remote tags yet.

Reply:
FWIW this was a quick copy paste job from me, so I take all the blame.

Reply:
In fairness if I steal your code I have to take the bad with the good :) have updated this to `tags_matching`.